tests/test_trace.py

```diff
@@ -111,3 +113,32 @@ def test_stop_clears_all_state(trace_exporter):
 
     # Calling export after stop
     trace_exporter.export(())
+
+
+def test_default_exporter_values(insert_key):
+    exporter = NewRelicTraceExporter(insert_key, service_name="Python Application")
+
+    assert isinstance(exporter._transport, DefaultTransport)
+    assert exporter.client._pool.host == SpanClient.HOST
+    assert exporter.client._pool.port == 443
+
+
+class CustomTransport(DefaultTransport):
+    pass
+
+
+def test_override_exporter_values(insert_key):
+    host = "non-default-host"
+    CustomTransport = type("CustomTransport", (DefaultTransport,), {})
```

Comment:
This looks to be redundant to the `CustomTransport` above. Any reason to define `CustomTransport` again here?
```suggestion
```

Reply:
Good catch, no reason at all 😄 I removed it.